core-service/src/modules/public-service/repository/mysql/mysql_public_service.go

```diff
@@ -107,8 +107,8 @@ func (m *mysqlPublicServiceRepository) Fetch(ctx context.Context, params *domain
 }
 
 func (m *mysqlPublicServiceRepository) MetaFetch(ctx context.Context, params *domain.Request) (total int64, lastUpdated string, err error) {
-	total, _ = m.count(ctx, ` SELECT COUNT(1) FROM public_services `)
-
+	query := ` SELECT COUNT(1) FROM public_services WHERE category=? ` // default list public service using filter by category
```

Comment:
```suggestion
	query := ` SELECT COUNT(1) FROM public_services WHERE category= ? ` // default list public service using filter by category
```

minor space after `=`

Reply:
done here https://github.com/jabardigitalservice/portal-jabar-services/pull/298/commits/3145f42a0a10f0350baab01295baf8b738089925